packages/shared/src/utils/config/config.ts

```diff
@@ -126,7 +126,7 @@ export const checkAndSetEndpointFromUrl = () => {
 };
 
 export const getDebugServiceWorker = () => {
-    const debug_service_worker_flag = window.localStorage.getItem('debug_service_worker');
+    const debug_service_worker_flag: string | null = window.localStorage.getItem('debug_service_worker');
```

Comment:
```suggestion
    const debug_service_worker_flag = window.localStorage.getItem('debug_service_worker');
```
@niloo-fs this type is unnecessary here since it's inferred from getItem() return value 🙂